include/net/fota_download.h

```diff
@@ -57,6 +57,8 @@ enum fota_download_error_cause {
 	FOTA_DOWNLOAD_ERROR_CAUSE_INVALID_UPDATE,
 	/** Actual firmware type does not match expected. Retry will not help. */
 	FOTA_DOWNLOAD_ERROR_CAUSE_TYPE_MISMATCH,
+	/** Downloading the update failed. The download may be retried. */
+	FOTA_DOWNLOAD_ERROR_CAUSE_CONNECTION_LOST,
```

Comment:
@chmielewskiandreas Wasn't this new even supposed to be removed, according to previous discussion? It doesn't seem to be set anywhere.

Reply:
Right, I can remove this to avoid dead code